### PR TITLE
[10.0][IMP] Add both pivot and graph views to account_due_list

### PR DESCRIPTION
**Pull request by robyf70**
Just to improve the account due list representation and pivoting

### CODE REVIEW

**Review by jackjack82 — APPROVED** [review]
fine for me

**Review by feketemihai — APPROVED** [review]

**Comment by agb80** [comment]
:+1: pretty usefull and easy improvement

**Comment by pedrobaeza** [comment]
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

And where are the definition of those views?

**Comment by robyf70** [comment]
@pedrobaeza The views aren't really necessary, defaults are pretty ok for me. Will increase the version number right now

**Comment by pedrobaeza** [comment]
But you should create them with the same defaults, or this module will not be inheritable as dependent modules might create other conflicting views instead of inheriting the main one.

**Comment by robyf70** [comment]
@pedrobaeza I've just pushed a new rebased patch with related pivot and graph views

**Comment by robyf70** [comment]
@pedrobaeza Sorry! Wrong patch, will resubmit

**Comment by robyf70** [comment]
@pedrobaeza Now looks fine

File: account_due_list/views/payment_view.xml [hunk 1]
```diff
@@ -94,18 +94,44 @@
...
         <field name="view_type">form</field>
-        <field name="view_mode">tree,form</field>
+        <field name="view_mode">tree,form,pivot,graph</field>
         <field name="context">{'search_default_from_invoices': 1}</field>
         <field name="view_id" ref="view_payments_tree"/>
         <field name="search_view_id" ref="view_payments_filter"/>
         <field name="domain">[('account_id.internal_type', 'in', ['receivable', 'payable'])]</field>
     </record>
 
+    <record model="ir.actions.act_window.view" id="view_payments_tree_action">
```
Comment (pedrobaeza) [inline account_due_list/views/payment_view.xml]: Not needed

File: account_due_list/views/payment_view.xml [hunk 2]
```diff
@@ -94,18 +94,44 @@
...
+        <field name="sequence" eval="1" />
+        <field name="view_mode">tree</field>
+        <field name="view_id" ref="view_payments_tree"/>
+        <field name="act_window_id" ref="action_invoice_payments"/>
+    </record>
+
+    <record model="ir.actions.act_window.view" id="view_payments_pivot_action">
```
Comment (pedrobaeza) [inline account_due_list/views/payment_view.xml]: Not needed.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And what about graph view?

**Comment by robyf70** [comment]
@pedrobaeza ok! I've done some changes and added the missing graph view, let me know if now looks ok for you

**Review by pedrobaeza — APPROVED** [review]

**Review by adrienpeiffer — APPROVED** [review]
👍

**Comment by pedrobaeza** [comment]
Cherry-picked for v11 in b2addc3

**Pull request merged**